### PR TITLE
Envirosuit fixes and replacement envirohelm removal

### PR DESCRIPTION
**Pull request by BrumalLake**
## About The Pull Request
The description on the security envirohelmet says that it provides flash protection passively so it will be removed from its description
Replacement envirohelms currently don't have any disadvantages not caused by bugs other than the curator's having a welding screen so they will be removed
Atmos envirosuits pieces also inherit radiation protection from engineer envirosuits.
Also some grammatical tweaks with envirohelmets including the removal of the word "plasma" which was inconsistently in some envirohelmet names.

## Why It's Good For The Game
Text on items not matching their actual functions is confusing. I remember being new and thinking "plasma envirosuits" were supposed to be clothing you wore while standing in plasma.

## Changelog
:cl:
tweak: made atmos envirosuits inherit radiation shielding from engineer envirosuits
del: replacement envirosuit helmets
spellcheck: fixed a few typos and false info in envirohelmet descriptions and made names linear
/:cl:

### CODE REVIEW

**Comment by bloons3** [comment]
Not even sec officers get _free_ and passive flash protection.

Plasmemes can use sechuds like everyone else for their flash protection.

**Comment by BrumalLake** [comment]
> 
> 
> Not even sec officers get _free_ and passive flash protection.
> 
> Plasmemes can use sechuds like everyone else for their flash protection.

advanced sunglasses are easily obtained and already a granted for sec; they should come with officers at this point
its listed in their description and they have little armor

the only thing this would prevent is an unknowing officer getting stomped by a swarm of revs with flashes after joining midround which is bull anyways

**Comment by PowerfulBacon** [comment]
The description should be updated to remove the part saying they get flash protection, the reason they don't have it is so that non plasmamemes cant get the helmet for free flash protection. Plasmameme security flash protection comes from the glasses they wear underneath.

**Comment by PowerfulBacon** [comment]
This code looks like its just going to give 0 flash protection while down and -1 flash protection while up. Nevermind all plasmaman helmets come with 2 flash protection.

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -355,6 +355,16 @@ BLIND     // can't see anything
...
+		
+/obj/item/clothing/head/helmet/space/plasmaman/security/visor_toggling() //security envirohelmets get flash protection without visor
+	up = !up
+	clothing_flags ^= visor_flags
+	flags_inv ^= visor_flags_inv
+	icon_state = "[initial(icon_state)]"
+	if(visor_vars_to_toggle & VISOR_FLASHPROTECT)
+		flash_protect = initial(flash_protect) - up
+	if(visor_vars_to_toggle & VISOR_TINT)
+		tint ^= initial(tint)
```
Comment (PowerfulBacon) [inline code/modules/clothing/clothing.dm]: Flash protection should come from glasses not the helmet itself due to free flash protection for getting the helmet. Description of the helmet should be updated to remove the flash protection instead.

**Comment by BrumalLake** [comment]
> 
> 
> The description should be updated to remove the part saying they get flash protection, the reason they don't have it is so that non plasmamemes cant get the helmet for free flash protection. Plasmameme security flash protection comes from the glasses they wear underneath.

but this doesnt apply to replacement ones just ones that spawn with the plasmaman
if you're looting a sec plasmaman they probably already have hud glasses for you to take for flash protection
plus if the normal helmet didnt have flash protection there would be no disadvantage or difference from replacement helmets other than the bug they currently have thats being fixed making them bloat

**Comment by PowerfulBacon** [comment]
> 
> 
> > The description should be updated to remove the part saying they get flash protection, the reason they don't have it is so that non plasmamemes cant get the helmet for free flash protection. Plasmameme security flash protection comes from the glasses they wear underneath.
> 
> but this doesnt apply to replacement ones just ones that spawn with the plasmaman
> if you're looting a sec plasmaman they probably already have hud glasses for you to take for flash protection
> plus if the normal helmet didnt have flash protection there would be no disadvantage or difference from replacement helmets other than the bug they currently have thats being fixed making them bloat

If its important that the replacement suits have a disadvantage give them less armor

**Comment by BrumalLake** [comment]
> 
> 
> > > The description should be updated to remove the part saying they get flash protection, the reason they don't have it is so that non plasmamemes cant get the helmet for free flash protection. Plasmameme security flash protection comes from the glasses they wear underneath.
> > 
> > 
> > but this doesnt apply to replacement ones just ones that spawn with the plasmaman
> > if you're looting a sec plasmaman they probably already have hud glasses for you to take for flash protection
> > plus if the normal helmet didnt have flash protection there would be no disadvantage or difference from replacement helmets other than the bug they currently have thats being fixed making them bloat
> 
> If its important that the replacement suits have a disadvantage give them less armor

the special protection envirohelms only give 10 more radiation protection or melee protection compared to the normal helmets, not a big disadvantage
only the curator's replacement has a disadvantage being that it has no welding screen

**Comment by BrumalLake** [comment]
I'll probably just remove the welding screens and lights from replacement helmets when I get around to it

**Comment by BrumalLake** [comment]
Will come back later, have better things to port

**Pull request closed without merging**